SimpleITK/utilities/tf.py

```diff
@@ -0,0 +1,118 @@
+# ========================================================================
+#
+#  Copyright NumFOCUS
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+# ========================================================================
+
+from pathlib import Path
+import SimpleITK as sitk
+import tensorflow as tf
+import numpy as np
+
+
+def _sitk_image_to_tensor(image: sitk.Image, name=None, channels=3) -> tf.Tensor:
+    """Convert an SimpleITK Image to a Tensorflow Tensor.
+
+    :param image: The SimpleITK Image to convert
+    :param name: The name of the resulting Tensor
+    :param channels: The number of channels to use for the resulting Tensor (default: 3)
+    :return: A Tensorflow Tensor
+
+    """
+
+    image = sitk.RescaleIntensity(image, outputMinimum=0.0, outputMaximum=255.0)
+    image = sitk.Cast(image, sitk.sitkUInt8)
```

Comment:
I think these two rows should be removed (rescale+cast). The DICOM images can be color.